app/components/stories/components/DownloadList.stories.mdx

```diff
@@ -0,0 +1,96 @@
+import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs/blocks'
+import DownloadList from '../../ch/components/DownloadList.vue'
+
+<Meta
+  title="Components/DownloadList"
+  component={DownloadList}
+/>
+
+export const Template = (args, { argTypes }) => ({
+  props: Object.keys(argTypes),
+  components: { DownloadList },
+  template: '<ul><DownloadList v-for="item in content" id="1" :title="item.title" :description="item.description" :url="item.url" :type="item.type" :date="item.date"  /></ul>'
+})
+
+
+# DownloadList
+
+<Canvas>
+  <Story
+    name="Example"
+    args={{
+      content: [
+        { title:"Information on the usage of websites", description:"Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean feugiat augue eu purus luctus rhoncus. Donec ultricies venenatis nibh, vel placerat est accumsan quis. Maecenas urna nibh, pretium pretium odio id, rhoncus rhoncus lorem. Nulla eu neque sagittis, cursus purus eget, sodales est. Duis at ultrices odio, ac egestas dolor.", url:"https://www.w3.org/WAI/ER/tests/xhtml/testfiles/resources/pdf/dummy.pdf", type:"PDF", date:"13.01.2022" },
+        { title:"Information on the usage of websites", description:"", url:"https://www.w3.org/WAI/ER/tests/xhtml/testfiles/resources/pdf/dummy.pdf", type:"JPEG", date:"22.01.2022" },
+      ]
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+# Interaction and accessibility
+
+Each DownloadList item is it's own button, they  can also be added within a list and their width is contained by the outer parent element.
+
+```html
```

Comment:
It's a bit dangerous to write manually markup here. Chances are big that we forget to update it if we change something in the component. As the HTML is already in the Storybook/Canvas tab, I would suggest to remove it.